evap/staff/tests/test_views.py

```diff
@@ -24,7 +24,10 @@
 
 
 def helper_delete_all_import_files(user_id):
-    file_filter = generate_import_filename(user_id, "*")
+    class AllFiles:
+        value = "*"  # used for globbing all existing files in tests.
+
+    file_filter = generate_import_filename(user_id, AllFiles)
```

Comment:
I like this clever approach, but to propose a more “traditional” approach: You could iterate over the whole `ImportType` enum and remove the corresponding file if it exists. You probably would also get ride of passing it to `glob`

Reply:
Yup, makes sense, now that we can actually iterate the enum.